torcharrow/functional.py

```diff
@@ -121,6 +123,7 @@ def add_tokens(
     """
     Append or prepend a list of tokens/indices to a column.
     This is a common operation to add EOS and BOS tokens to text.
+    Note: this method is only available if torcharrow is built from source with the `USE_TORCH` flag enabled.
```

Comment:
Isn't this method in general available? (it doesn't require any thing from libtorch)

Reply:
Yup you're right. I moved the corresponding UDF out of the `ifdef` statement so that this method is available regardless of if TA is built with torch. Thanks for the catch!